chapters/infrastructure/linters/reading/README.md

```diff
@@ -1,14 +1,37 @@
-### Linters
+# Linters
 
-Owner: Gabi Mocanu
-Reviewer: Alex Apostolescu
+Linters are tools for checking consistency and quality of code, but they can be used on every type of content.
+Enforce a consistent style for writing content, making it easier to read and maintain for content creators.
+There are different types of linters, depending on the type of content.
+For example, a linting tool for code is different from a linting tool for text.
```

Comment:
This does not add anything. You could talk about differences between linters when discussing rules

Reply:
It's an introduction to the subject for people who are not familiar with this.
I think it's pretty simple and worth having.